misc/emacs/dbscheme-mode.el

```diff
@@ -0,0 +1,27 @@
+
+(define-generic-mode
+    'dbscheme-mode ; mode name
+
+  ;; comments
+  '(("//" . nil)
+    ("/*" . "*/"))
+
+  ;; keywords
+  '("case" "ref" "unique" "of")
+
+  ;; other things to highlight
+  `((,ql/primitive-type-regex . 'font-lock-type-face)
```

Comment:
`ql/primitive-type-regex` is not defined.
We use the double-dash convention for naming internal variables: use `ql--primitive-type-regex` instead.
Ditto below.

Reply:
I also prefer that convention, updated.